101-app-configuration-store-kv/azuredeploy.json

```diff
@@ -0,0 +1,55 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "configStoreName": {
+            "type": "string",
+            "metadata": {
+                "description": "Specifies the name of the App Configuration store."
+            }
+        },
+        "keyValueName": {
+            "type": "string",
+            "defaultValue": "myKey$myLabel",
+            "metadata": {
+                "description": "Specifies the name of the key-value resource. The name is a combination of key and label with $ as delimiter."
+            }
+        }
+    },
+    "resources": [
+        {
+            "name": "[parameters('configStoreName')]",
+            "type": "Microsoft.AppConfiguration/configurationStores",
+            "apiVersion": "2020-07-01-preview",
+            "location": "[resourceGroup().location]",
+            "sku": {
+                "name": "standard"
+            },
+            "resources": [
+                {
+                    "name": "[parameters('keyValueName')]",
+                    "type": "keyValues",
+                    "apiVersion": "2020-07-01-preview",
+                    "dependsOn": [
+                        "[parameters('configStoreName')]"
+                    ],
+                    "properties": {
+                        "value": "Hello World!",
```

Comment:
these other properties should be parameters as well

Reply:
I'm trying to make this template as neat as possible. I'd love to follow it if `all values should be parameters` is a convention.

Reply:
I don't think you'll see that convention anywhere - what you will see are functional samples.  Generally when someone wants to create a key/value pair, the key and value unique to the scenario.  The parameters can use defaultValues to steamline the "hello world" scenario